src/tables.ts

```diff
@@ -586,6 +718,56 @@ export class Tables {
     return clefs[clef];
   }
 
+  /** Use the provided key to look up a FontInfo in CommonMetrics. **/
+  static lookupMetricFontInfo(key: string): Required<FontInfo> {
+    return {
+      family: Tables.lookupMetric(`${key}.fontFamily`),
+      size: Tables.lookupMetric(`${key}.fontSize`),
+      weight: Tables.lookupMetric(`${key}.fontWeight`),
+      style: Tables.lookupMetric(`${key}.fontStyle`),
```

Comment:
@rvilarl I'm going to merge this PR
... but in your next PR consider doing `key + '.fontFamily'`. It's possible that it minimizes better. :-) You can check the output in our build/ dir, haha.